sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -20,5 +21,28 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ResourceId => GetRecordedVariable("RESOURCE_ID");
         public string WorkspacePrimaryResourceId => GetRecordedVariable("WORKSPACE_PRIMARY_RESOURCE_ID");
         public string WorkspaceSecondaryResourceId => GetRecordedVariable("WORKSPACE_SECONDARY_RESOURCE_ID");
+        public Uri LogsEndpoint => new(GetEndpoint());
+
+        private Dictionary<string, string> regions = new Dictionary<string, string>()
+        {
+            { "AzureCloud", "https://api.loganalytics.io/v1" },
```

Comment:
Is it not possible to pull this down via the bicep file?

Reply:
No I don't believe so. Other languages are pulling from the test environment as well

Reply:
So the service doesn't expose these endpoints programmatically? Users have to look at docs?

Reply:
Not sure about monitor in particular but a lot of services have this hard coded and expect people to discover this through documentation.

Reply:
I am curious about v1 in the end - Does the service have v2 endpoints too? Is this their versioning scheme?

Reply:
Yes the service is in v1 right now and v2 I'm assuming will come out in the future.